src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -209,6 +209,23 @@ def load_python_module(content, source=None):
         ) from e
 
 
+def load_function_from_function_path(func_path: str):
+    """Load a function from a function path.
+
+    The function path should be in the format of "module_name.function_name".
+    """
+    try:
+        module_name, func_name = func_path.rsplit(".", 1)
+        module = importlib.import_module(module_name)
+        f = getattr(module, func_name)
+        if callable(f):
+            return f
+        else:
+            raise FunctionPathValidationError(f"'{f}' is not callable.")
+    except Exception as e:
+        raise FunctionPathValidationError(f"Failed to parse function from function path: '{func_path}' with error: {e}")
```

Comment:
Unable to display list of items due to XXX. The list function needs to follow the format "module_name.function_name" to work properly. Please contact the tool author/support team for troubleshooting assistance.

Reply:
updated